opencti-platform/opencti-front/src/private/components/analysis/malware_analyses/MalwareAnalysis.tsx

```diff
@@ -0,0 +1,155 @@
+  },
+}));
+
+const MalwareAnalysisFragment = graphql`
+    fragment MalwareAnalysis_malwareAnalysis on MalwareAnalysis {
+        id
+        standard_id
+        entity_type
+        x_opencti_stix_ids
+        created
+        modified
+        created_at
+        revoked
+        confidence
+        product
+        submitted
+        result_name
+        createdBy {
+            ... on Identity {
+                id
+                name
+                entity_type
+            }
+        }
+        creators {
+            id
+            name
+        }
+        objectMarking {
+            edges {
+                node {
+                    id
+                    definition_type
+                    definition
+                    x_opencti_order
+                    x_opencti_color
+                }
+            }
+        }
+        objectLabel {
+            edges {
+                node {
+                    id
+                    value
+                    color
+                }
+            }
+        }
+        objectAssignee {
+            edges {
+                node {
+                    id
+                    name
+                    entity_type
+                }
+            }
+        }
+        status {
+            id
+            order
+            template {
+                name
+                color
+            }
+        }
+        workflowEnabled
+        ...MalwareAnalysisDetails_malwareAnalysis
+    }
+`;
+
+const MalwareAnalysis: FunctionComponent<{ data: MalwareAnalysis_malwareAnalysis$key }> = ({ data }) => {
+  const classes = useStyles();
+  const malwareAnalysisFragment = useFragment(MalwareAnalysisFragment, data);
+
+  return (
+      <div>
+          <StixDomainObjectHeader
+              stixDomainObject={malwareAnalysisFragment}
+              PopoverComponent={<MalwareAnalysisPopover id={malwareAnalysisFragment.id}/>}
+          />
+          <Grid
+              container={true}
+              spacing={3}
+              classes={{ container: classes.gridContainer }}
+          >
+              <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
+                  <MalwareAnalysisDetails malwareAnalysisFragment={malwareAnalysisFragment} />
+              </Grid>
+              <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
+                  <StixDomainObjectOverview stixDomainObject={malwareAnalysisFragment} />
+              </Grid>
+          </Grid>
+          <Grid
+              container={true}
+              spacing={3}
+              classes={{ container: classes.gridContainer }}
+              style={{ marginTop: 25 }}
+          >
+              <Grid item={true} xs={6}>
+                  <SimpleStixObjectOrStixRelationshipStixCoreRelationships
+                      stixObjectOrStixRelationshipId={malwareAnalysisFragment.id}
+                      stixObjectOrStixRelationshipLink={`/dashboard/analysis/malware_analyses/${malwareAnalysisFragment.id}/knowledge`}
```

Comment:
Can you make a const for the url to have a readable line ?

Reply:
Done!